test/test_torch.py

```diff
@@ -378,7 +378,21 @@ def _digamma_input(self, test_poles=True):
     @unittest.skipIf(not TEST_SCIPY, "Scipy not found")
     def test_digamma(self):
         from scipy.special import digamma
-        self._test_math(torch.digamma, digamma, self._digamma_input())
+
+        # scipy 1.1.0 changed when it returns +/-inf vs. NaN
```

Comment:
nit: It would be nice to put a little more context here, something like "digamma is undefined at certain values and scipy 1.1.0 changed its behavior from returning +/- inf at those values to NaN"